src/scripts/main.js

```diff
@@ -1,3 +1,69 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const fieldBlock = document.querySelector('.field tbody');
+let field = Array.from(fieldBlock.querySelectorAll('tr'));
```

Comment:
The `field` variable is initialized before the `DOMContentLoaded` event listener. This might cause issues if the script is executed before the DOM is fully loaded. Consider moving this initialization inside the `DOMContentLoaded` event listener to ensure the DOM elements are available.